src/components/banner/Banner.vue

```diff
@@ -1,102 +1,336 @@
+      <div class="video-controls video" v-if="media_type === 'video'">
+        <button class="video-btn video-btn__pause" id="video-btn-pause" v-if="enable_autoplay === true">
+          <span class="element-invisible">Pause</span>
+        </button>
+        <button class="video-btn video-btn__play" id="video-btn-play" v-if="enable_autoplay === false">
+          <span class="element-invisible">Play</span>
+        </button>
+      </div>
+
     </div>
     <div class="banner__container">
       <div class="banner__content">
-        <slot name="headline">
+        <header class="banner__title" v-if="$slots.title || title">
+          <div :class="[headlineClasses, 'banner__pre-title', 'headline']"><slot name="pre_title">{{ pre_title }}</slot></div>
           <uids-headline
-            v-if="title"
-            :level="getHeadlineSettings.level"
-            :class="getHeadlineSettings.classes"
-            :href="url"
-          >{{ title }}</uids-headline>
-        </slot>
-        <slot></slot>
-        <uids-button
-          v-if="url && button_text"
-          :href="url"
-          class="bttn--secondary bttn--caps"
-          :arrow="true"
-        >{{ button_text }}</uids-button>
+            :text_style="headline_style"
+            :class="headlineClasses"
+          >
+            <a v-if="headlineLink" :href="headlineLink" class="click-target">
+              <slot name="title">{{ title }}</slot>
+            </a>
+            <template v-else>
+              <slot name="title">{{ title }}</slot>
+            </template>
+          </uids-headline>
+        </header>
+
+        <div class="banner__text" v-if="content" >
+          <slot name="content">{{ content }}</slot>
+        </div>
+
+        <footer class="banner__action" v-if="button_label || button_label_2 || button_label_3" >
+          <!-- Render buttons when multiple URLs are present -->
+          <div v-if="url && (url_2 || url_3)" class="bttn--row">
+            <uids-button
+              :class="['bttn', ...buttonClasses]"
+              :url="url"
+              :no_default_classes="true"
+              size="medium"
+              v-if="url && button_label">
+              <slot name="button_label">{{ button_label }}</slot>
+              <slot name="button_icon"></slot>
+            </uids-button>
+
+            <uids-button
+              :class="['bttn', ...buttonClasses]"
+              :url="url_2"
+              :no_default_classes="true"
+              size="medium"
+              v-if="url_2 && button_label_2">
+              {{ button_label_2 }}
+              <slot name="button_icon"></slot>
+            </uids-button>
+
+            <uids-button
+              :class="['bttn', ...buttonClasses]"
+              :url="url_3"
+              :no_default_classes="true"
+              size="medium"
+              v-if="url_3 && button_label_3">
+              {{ button_label_3 }}
+              <slot name="button_icon"></slot>
```

Comment:
```suggestion
```
I don't think we need a slot for this. We can leave it to the implementer to decide to include a button icon if desired and allow the component to be more flexible and less opinionated.